lib/mayaUsd/ufe/UsdUIInfoHandler.cpp

```diff
@@ -68,6 +69,46 @@ void addMetadataCount(
         addMetadataStrings(refs.size(), tooltip, needComma, singular, plural);
     }
 }
+
+std::vector<double>& getInvisibleColor()
+{
+    static std::vector<double> rgb;
+    return rgb;
+}
```

Comment:
Great this is much better. Just one question. What is the reasoning for a static variable and helpers instead of a member var and private methods?

Reply:
None. Just that the code evolved and this was the least amount of changes. OTOH, there is no reason to have the RGB value per-instance as the color is global, so that could be a post-hoc justification :).

Reply:
There will always only be a **single** instance of this handler class. It is created and registered by Global.cpp and when someone asks for it, you get back the shared_ptr. So please use a member var to store the value and add private member functions to call the Maya command to get/update the value.